crates/holochain/Cargo.toml

```diff
@@ -187,11 +184,43 @@ only_check_wasms = ['holochain_wasm_test_utils/only_check']
 
 # Enables at-rest encryption of the SQLite database.
 # Incompatible with "no-deps".
-db-encryption = [ "holochain_keystore/db-encryption", "holochain_sqlite/db-encryption" ]
+db-encryption = [
+  "rusqlite/bundled-sqlcipher-vendored-openssl",
+  "holochain_keystore/db-encryption",
+  "holo_hash/db-encryption",
+  "holochain_cascade/db-encryption",
+  "holochain_conductor_api/db-encryption",
+  "holochain_keystore/db-encryption",
+  "holochain_p2p/db-encryption",
+  "holochain_sqlite/db-encryption",
+  "holochain_state/db-encryption",
+  "holochain_types/db-encryption",
+  "holochain_zome_types/db-encryption",
+  "kitsune_p2p/db-encryption",
+  "kitsune_p2p_types/db-encryption",
+  "kitsune_p2p_block/db-encryption",
+  "kitsune_p2p_bootstrap/db-encryption",
+]
 
 # Compile SQLite from source rather than depending on a library.
 # Incompatible with "db-encryption"
-no-deps = [ "holochain_keystore/no-deps", "holochain_sqlite/no-deps" ]
+no-deps = [
```

Comment:
Do you think the ship has sailed on renaming this? We could add a stub `no-deps` feature to the crates that previously had it, and introduce a new `sqlite-bundled` feature.

Reply:
Anywhere that's new can definitely have a different name. I think maintaining compatibility on this PR is a good thing because I plan to backport it so I will stub as you suggest. But even that could be removed in the near future. I'm on this 😄

Reply:
I take back the compatibility part actually. I'm just going to make this change on develop and I'll have to adjust it when I backport anyway. I actually have to maintain the feature all the way down because we've just learned from the `0.1.4` release we have to maintain internal crate interfaces on release too so I can't rename features and expect not to have problems. I will layer a commit on top of this change when I go to backport to ensure it's not a breaking change.

Otherwise though I have renamed `no-deps` to `sqlite` because as-per the comment below about using a system dependency, this feature actually just enables sqlite functionality and there's no 'not bundled' option. So this keeps that feature simple. Then `db-encryption` has become `sqlite-encrypted` which hopefully makes it a bit more obvious that it's one or the other and they are not to be used together.

Reply:
I'm okay with this, as it represents our intention pretty well. But just note that it is totally okay in rusqlite to specify both features... `bundled-sqlcipher` is still `bundled` : )

Reply:
Ahh okay, I did not realise that. That helps answer the question below about defaults. Thank you